base/random.jl

```diff
@@ -50,8 +50,8 @@ else # !windows
         RandomDevice(unlimited::Bool=true) = new(open(unlimited ? "/dev/urandom" : "/dev/random"), unlimited)
     end
 
-    rand{T<:Union{Bool, Base.BitInteger}}(rd::RandomDevice, ::Type{T}) = read( rd.file, T)
-    rand!(rd::RandomDevice, A::Array{<:Union{Bool, Base.BitInteger}})  = read!(rd.file, A)
+    rand(rd::RandomDevice, ::Type{T}) where {T<:Union{Bool,Base.BitInteger}} = read( rd.file, T)
+    rand!(rd::RandomDevice, A::Array{<:Union{Bool,Base.BitInteger}})        = read!(rd.file, A)
```

Comment:
Looks like you'll need another space there before the `=` if you want it to continue to line up